### PR TITLE
GUVNOR-2696: Guided Decision Table: *.XLSX can not be converted to *.GDST

### PR DESCRIPTION
**Pull request by manstis**
See https://issues.jboss.org/browse/GUVNOR-2696

We simply did not validate a file's extension against either XLS or XLSX.

### CODE REVIEW

File: drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DecisionTableXLSToDecisionTableGuidedConverter.java [hunk 1]
```diff
@@ -163,7 +167,7 @@ public ConversionResult convert( final Path path ) {
         ConversionResult result = new ConversionResult();
 
         //Check Asset is of the correct format
-        if ( !xlsDTableType.accept( path ) ) {
+        if ( !( xlsDTableType.accept( path ) || xlsxDTableType.accept( path ) ) ) {
             result.addMessage( "Source Asset is not an XLS Decision Table.",
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DecisionTableXLSToDecisionTableGuidedConverter.java]: Just a minor thing, maybe also in the error message should be mentioned as XLS so XLSX extension.

Comment (manstis) [inline drools-wb-screens/drools-wb-dtable-xls-editor/drools-wb-dtable-xls-editor-backend/src/main/java/org/drools/workbench/screens/dtablexls/backend/server/conversion/DecisionTableXLSToDecisionTableGuidedConverter.java]: Good catch. Fixed.

**Comment by manstis** [comment]
@Rikkola Would you mind passing your eyes over this?

**Comment by Rikkola** [comment]
I like how you use the actual file to test. 
+1

**Pull request merged**